apps/interviewer/e2e/README.md

```
@@ -0,0 +1,78 @@
+
+# Run headed against your local Chromium for debugging — functional
+# assertions only, no baseline regeneration
+pnpm --filter @codaco/interviewer test:e2e:headed
+```
+
+`test:e2e:headed` may need a local Chromium binary installed once first:
+`pnpm --filter @codaco/interviewer exec playwright install chromium`.
+
+## Layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)